cmd/aperturectl/cmd/blueprints/list.go

```diff
@@ -21,11 +22,11 @@ var listCmd = &cobra.Command{
 	Short: "List Aperture Blueprints",
 	Long: `
 Use this command to list the Aperture Blueprints which are already pulled and available in local system.`,
-	Example: `aperturectl blueprints list
+	Example: fmt.Sprintf(`aperturectl blueprints list
```

Comment:
I don't think these formatted strings are needed. These are just examples and can be any version.

Reply:
My idea to have that is when user run --help, it shows the same version as aperturectl. We might end up where the aperturectl is moved a lot ahead in terms of version and the example keeps showing the older versions in example and someone using the example directly copy-pasting from docs.

Reply:
The version is blueprints from remote repo. There might be a point where we separate out `aperturectl` as a repo of its own.

Reply:
So passing `aperturectl`'s version here is pointless.

Reply:
Should we remove the version from example? I don't feel showing different versions for example on docs for docs and example won't look good? WDYT?

Reply:
Let's keep it there till it is under aperture repo.. will use some other way when it is taken out of it

Reply:
It is good to show what all the command is capable of.

Reply:
It clearly being an example, it should not matter what version is shown. It is under "Examples" section.